tasks/create.yaml

```diff
@@ -60,7 +60,17 @@ tasks:
     actions:
       - description: "Build the UDS Core Pepr Module"
         cmd: |
-          CUSTOM_PEPR_IMAGE=$( [ "${FLAVOR}" = "registry1" ] && echo "--custom-image ${REGISTRY1_PEPR_IMAGE}" ) || CUSTOM_PEPR_IMAGE=""
+          if [ "${FLAVOR}" = "registry1" ] ; then
```

Comment:
IMO we should let `PEPR_CUSTOM_IMAGE` override the pepr image even if the flavor is registry1.

Reply:
We don't feel too strongly about what should override what, so deferring to you!

PR updated to allow PEPR_CUSTOM_IMAGE to override FLAVOR below. :+1: